chain/client/src/info.rs

```diff
@@ -112,14 +115,26 @@ impl InfoHelper {
         } else {
             White.bold().paint("")
         };
-        info!(target: "stats", "{} {} {} {} {} {}",
-              Yellow.bold().paint(display_sync_status(&sync_status, &head, genesis_height)),
-              validator_info_log,
-              Cyan.bold().paint(format!("{:2}/{:?}/{:2} peers", network_info.num_active_peers, network_info.highest_height_peers.len(), network_info.peer_max_count)),
-              Cyan.bold().paint(format!("⬇ {} ⬆ {}", pretty_bytes_per_sec(network_info.received_bytes_per_sec), pretty_bytes_per_sec(network_info.sent_bytes_per_sec))),
-              Green.bold().paint(format!("{:.2} bps {}", avg_bls, gas_used_per_sec(avg_gas_used))),
-              Blue.bold().paint(format!("CPU: {:.0}%, Mem: {}", cpu_usage, pretty_bytes(memory_usage * 1024)))
-        );
+
+        if self.log_color {
+            info!(target: "stats", "{} {} {} {} {} {}",
+                  Yellow.bold().paint(display_sync_status(&sync_status, &head, genesis_height)),
+                  validator_info_log,
+                  Cyan.bold().paint(format!("{:2}/{:?}/{:2} peers", network_info.num_active_peers, network_info.highest_height_peers.len(), network_info.peer_max_count)),
+                  Cyan.bold().paint(format!("⬇ {} ⬆ {}", pretty_bytes_per_sec(network_info.received_bytes_per_sec), pretty_bytes_per_sec(network_info.sent_bytes_per_sec))),
+                  Green.bold().paint(format!("{:.2} bps {}", avg_bls, gas_used_per_sec(avg_gas_used))),
+                  Blue.bold().paint(format!("CPU: {:.0}%, Mem: {}", cpu_usage, pretty_bytes(memory_usage * 1024)))
+            );
+        } else {
+            info!(target: "stats", "{} {} {} {} {} {}",
```

Comment:
Instead of copy-pasting, I suggest we prepare the values first, then conditionally apply the colors to them, and then print them out.

With the current approach, it is very easy to introduce discrepancy between colored and non-colored output